### ISSUE TITLE
Yahoo、zozo買収、paypayモール

### ISSUE THREAD

**Issue by YusukeBBB** [cyber-s/blog#633]
ちょっと調べてみたら、PayPayすごかったんだな、と。戦略通りで

1回目の100億円キャンペーンで一気に認知度を高める
2回目は上限を設け、決済数を増やすことで、利便性を幅広く知らしめ、定着させる

zozoをここに（paypayモールに）ブチ込むことでシナジーを生む、、か
zozoのユーザーは若年層がメインだろうか。マーケットはでかけれど、ご老体を相手に頑張るより、若年層をyahooに取り込み、結果全体を伸ばす、みたいなのは正しい気が。しかし攻めるなー

決算関連資料
https://about.yahoo.co.jp/ir/jp/archives/earnings/

![Screenshot 2019-09-12 at 9 15 59 AM](https://user-images.githubusercontent.com/44961904/64746486-4ee79800-d53e-11e9-9870-1bdac620ee7d.png)
![Screenshot 2019-09-12 at 9 15 53 AM](https://user-images.githubusercontent.com/44961904/64746491-51e28880-d53e-11e9-8a2c-2e1ba80b3fe7.png)

**Comment by ks1108** [comment]
実はですね、僕が最近「購入回数セグメント」にハマっているのも笑PayPayに関する記事を読んだことがきっかけです。

彼らにとっての「成功」は、家電とかの高額決済で単発で使われることではなく、例えば公共料金とか、日常のコンビニ程度の買い物とかで「少額でもいいから頻繁に利用されて生活に浸透すること」なので、その分岐点となりえる2回めの利用を少額決済でもいいからと促したのが「2度めの100億キャンペーン」だったみたいです。

で、ECでも利用回数でその後の利用頻度・継続率が変わる分岐点があるだろうと思い、関連書籍を読んで、 F0,F1,F2...の考え方を知りました。うまく活用できるかデータをとったここからが本番ですが。

**Comment by ks1108** [comment]
前澤さんも持ち前の強運が通用するのここまでって感じなので、あとはホンモノの経営者にまかせて月から地球を見守っていればいいんじゃないかと思います。ZOZO株価まだまだあがりますね。

**Comment by YusukeBBB** [comment]
はい、この決算資料見て、思いました。ポイントなのはただ2回使わせたのではなく、同じキャンペーンアプローチに見えて1回目と2回目で手法がちょっと違うことですよね。同じやり方だとたぶんこうはなってなかったんだと思います。それが我々にとって何なのだろうってのが考えるべきことなのかと思いました

前澤さん何やるんでしょうね。夢追い人っぽいので、そういう人を支援するVCみたいのやってくれないかな。以前音楽系のビジネスアイデア思いついた時に、こういう人なら投資してくれそう、とか思ってましたw

**Comment by ks1108** [comment]
そのとおりだと思います。それを踏まえた上で、僕は浸透が難しいんじゃないかと思ったんですが、実利用のMAUがたぶん公開されてないのでわかりませんね。ここにZOZOをもってくるとは思っていませんでいした。公開買い付けのプレミアムは20％みたいです。
>ポイントなのはただ2回使わせたのではなく、同じキャンペーンアプローチに見えて1回目と2回目で手法がちょっと違うことですよね。
